tests/test_help_pages.py

```diff
@@ -11,10 +11,26 @@
 
 def traverse_commands(command, args):
     yield args
+
     if isinstance(command, click.Group):
         for name, sub in command.commands.items():
             yield from traverse_commands(sub, args + [name])
 
+        params = command.params
+        if params:
+            try:
+                context_choices = params[0].type.choices
```

Comment:
This is probably not the best indicator. We should "find" the param with the name "--type" in the list.